gradle.properties

```diff
@@ -5,7 +5,7 @@ kotlin.code.style=official
 # Note: testng.kotlin-library.gradle.kts adds kotlin-stdlib for testImplementation
 kotlin.stdlib.default.dependency=false
 
-testng.version=7.7.0
+testng.version=7.6.1
```

Comment:
Why not the latest version or the next one?

Reply:
The last released version was `7.6.0`. Am guessing that this qualifies to be a bugfix release. Hence the version as `7.6.1`

`7.7.0` - Will be the version for the next release (I have already moved the milestones etc., Dont want to go through that ritual once again)

Reply:
It would be nice if we release 7.6.x during this year and maybe once per year lift the major version to allow smaller releases from time to time.